spec/services/draw_starter_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe DrawStarter do
+  describe '.start' do
+    it 'calls :start on an instance of DrawStarter' do
+      draw = double('draw')
+      draw_starter = create_mock_draw_starter(draw: draw)
+      DrawStarter.start(draw: draw)
+
+      expect(draw_starter).to have_received(:start)
+    end
+  end
+
+  def create_mock_draw_starter(param_hash)
+    draw_starter = double('draw_starter')
```

Comment:
I *think* that `instance_spy` is generally the preferred mock method ([thoughtbot post](https://robots.thoughtbot.com/a-closer-look-at-test-spies)), though there are trade-offs. 

If this is kept as a standard mock, though, it should be a [verifying double](https://relishapp.com/rspec/rspec-mocks/v/3-5/docs/verifying-doubles) so it checks that the stubbed methods are present on the actual class/instance of said class